templates/security/cve/_cve-table.html

```diff
@@ -17,10 +17,10 @@
   </thead>
   <tbody>
     {% for cve in cves %}
-      {% for package_name, statuses in cve.status_tree.items() %}
+      {% for package_name, statuses in cve.packages.items() %}
```

Comment:
Love this change. This is much more straightforward. Thanks for massively improving on my bad `status_tree` name.

I'd in fact love it if we could rename this all the way down to the model at https://github.com/albertkol/ubuntu.com/blob/search-cves-by-component/webapp/security/models.py#L68. Do you think we could do that? You don't have to do it in this PR if you think it'll delay things.

Reply:
I will rename the `status_tree` method to `packages` as part of my second PR (https://github.com/canonical-web-and-design/ubuntu.com/pull/8090) to make my life easier.